kratos/mpi/tests/test_mpi_serializer.py

```diff
@@ -0,0 +1,111 @@
+import KratosMultiphysics
+import KratosMultiphysics.mpi as KratosMPI
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+import KratosMultiphysics.KratosUnittest as UnitTest
+import sys
+import os
+
+dependencies_are_available = kratos_utilities.CheckIfApplicationsAvailable("MetisApplication")
+if dependencies_are_available:
```

Comment:
`distributed_import_model_part_utility` is safe to import but will fail if you use metis
```suggestion

```

Reply:
You are right!